Issues/Week331.md

```diff
@@ -3,7 +3,7 @@
 
 * [Open app in specific view when push notification is tapped (iOS 13+)](https://fluffy.es/open-app-in-specific-view-when-push-notification-is-tapped-ios-13/), by [@soulchildpls](https://twitter.com/soulchildpls)
 * [Codable Conformance for Swift Enums with Multiple Associated Values of Different Types](https://lostmoa.com/blog/CodableConformanceForSwiftEnumsWithMultipleAssociatedValuesOfDifferentTypes/), by [@lostmoa_nz](https://twitter.com/lostmoa_nz)
-* [Conquering Capture LIsts](https://scotteg.github.io/Conquering-Capture-Lists), by [@scotteg](https://twitter.com/scotteg)
+* [Capture LIsts](https://scotteg.github.io/capture-lists), by [@scotteg](https://twitter.com/scotteg)
```

Comment:
`* Capture LIsts`: shoudln't it say `Capture Lists` (lower case `i`)? 👀

Reply:
@zntfdr Fixed, thank you 🙏